Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -359,11 +359,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 $this->fixer->setLinter(new Linter($config->getPhpExecutable()));
             } catch (UnavailableLinterException $e) {
                 if ($configFile && 'txt' === $input->getOption('format')) {
-                    $output->writeln('Unable to use linter, can not find PHP executable');
+                    $output->writeln(sprintf('Unable to use linter, can not find PHP executable "%s".', $config->getPhpExecutable()));
```

Comment:
`ConfigInterface->phpExecutable` is null by default, so the message will be
`Unable to use linter, can not find PHP executable ""`.
Looks odd